<!DOCTYPE html>
<html>
	<head>
		<link rel="stylesheet" href="reset.css" />
		<link rel="stylesheet" href="style.css" />
		<title>Shoichi Yip personal page</title>
	</head>
	<body>
		<div class="main">
			<h1>Shoichi Yip</h1>
			<div class="section">
				<p>I am a PhD candidate in Physics supervised by Clément Nizak and Olivier Rivoire on the topic of <em>High-throughput experiments and statistical data modeling for the engineering of enzyme specificity</em>. I work at the crossroads of biophysics, biochemistry, bioinformatics, molecular biology and machine learning.</p>
				<ul>
					<li><a href="https://www.labojeanperrin.fr/?cnizak">EvoMorph Team</a> at Laboratoire Jean Perrin (Sorbonne Université)</li>
					<li><a href="https://statbio.net">StatBio Team</a> at Gulliver Lab (ESPCI Paris)</li>
				</ul>
			</div>
			<div class="section">
				<h2>Research</h2>
				<p>Currently, I am working on a project that aims at modeling <strong>genotype-phenotype relations in biomolecules</strong>. In particular, my goal is to explain how biophysical properties emerge as a result of complex interactions between components of these systems (e.g. long-range interactions between amino acidic residues in proteins).</p>
<p>The model system of c	hoice is a family of proteolytic enzymes called <strong>serine protease</strong> [1], which has been studied for a long time by biochemists but its internal mechanisms have not yet been well understood. The biophysical property of interest is their <strong>specificity profile</strong>, which is heterogeneous and cannot be exhaustively explained just in terms of geometrical / chemical properties (i.e. short-range interactions) [2].</p>
<p>I use a set up that a	llows for <strong>high-throughput</strong> [3] <strong>and low-throughput measurements</strong>, that combines tools from bioinformatics, droplet microfluidics and molecular biology. These data can in turn be used to train <strong>machine-learning models</strong> that can capture coevolutionary signals and are also able to generate artificial proteins with the desired biophysical properties [4, 5].</p>
				<ul class="resref">
					<li>[1] Hedstrom, Lizbeth. "Serine protease mechanism and specificity." Chemical reviews 102.12 (2002): 4501-4524.</li>
					<li>[2] Gr&aacute;f, L&aacute;szl&oacute;, et al. "Electrostatic complementarity within the substrate-binding pocket of trypsin." Proceedings of the National Academy of Sciences 85.14 (1988): 4961-4965.</li>
					<li>[3] Romero, Philip A., Tuan M. Tran, and Adam R. Abate. "Dissecting enzyme function with microfluidic-based deep mutational scanning." Proceedings of the National Academy of Sciences 112.23 (2015): 7159-7164.</li>
					<li>[4] Russ, William P., et al. "An evolution-based model for designing chorismate mutase enzymes." Science 369.6502 (2020): 440-445.</li>
					<li>[5] Halabi, Najeeb, et al. "Protein sectors: evolutionary units of three-dimensional structure." Cell 138.4 (2009): 774-786.</li>
				</ul>
				</div>
				<div class="section">

				<h2>Teaching</h2>
				<ul>
					<li>Pratique Expérimentale et Numérique - M1 Physique et Mécanique - Sorbonne Université (UE UM4MEP10 / Feb 2026) <a href="./brownian/index.html">Fiche de TP</a></li>
				</ul>
			</div>
			<div class="section">
				<h2>Links</h2>
				<ul>
					<li><a href="https://www.labojeanperrin.fr/?syip">LJP - Personal Page</a></li>
					<li><a href="https://theses.fr/s418594">Theses.fr page</a></li>
					<li><a href="https://orcid.org/0009-0000-1420-3982">ORCiD page</a></li>
				</ul>
			</div>
		</div>
	</body>
</html>
